target/linux/ath79/dts/ar9331_etactica_eg200.dts

```diff
@@ -106,7 +106,7 @@
 				reg = <0x50000 0xfa0000>;
 			};
 
-			art: art@ff0000 {
+			art@ff0000 {
```

Comment:
Should all partitions be called `partition@x`?

Reply:
I really don't want to make too many unrelated changes. with this for example, notice how label is missing.